addons/ui/org.openhab.ui.cometvisu/readme.md

```diff
@@ -0,0 +1,195 @@
+Documentation of the CometVisu Backend for openHAB2
+
+## Introduction
+
+This adds a backend for the web based visualization CometVisu (http://www.cometvisu.org). The CometVisu is a highly customizable visualization, that runs in any browser. Despite the existing browser based UI´s in openHAB, the CometVisu does not rely on sitemaps (although they can be used). The layout is defined with an XML-based configuration file, which allows a multi column layout.
+
+This is just a short overview, more details will be added soon!
+
+## Requirements
+
+* openHAB 2.0 or greater<br>
+* CometVisu >=0.8.6? (not released yet) or greater (https://github.com/CometVisu/CometVisu/releases).<br>
+
+## Installation
+
+* Copy the addon org.openhab.ui.cometvisu*.jar to the openHAB addon folder
+* Then just extract the "release" folder (the one which contains the index.html file) of the downloaded CometVisu archive in the runtime folder off your openHAB2 server. 
```

Comment:
Let's discuss, what options we have here: With the new Karaf-based distro, it should be possible to install add-ons automatically without the need of manual extractions steps. The only files besides the bundle should be files for the conf folder.

Would it be possible to package the "web" folder within the bundle and register it directly in the HTTPService? Or does the user need to be able to do (many?) changes to these files?
If it is a single file that he might have to edit, this could potentially be places in the conf folder and served by a servlet under an appropriate url. What do you think?